src/content/docs/new-relic-one/use-new-relic-one/get-started/introduction-new-relic-one.mdx

```diff
@@ -81,21 +72,41 @@ We provide multiple ways to understand dependencies, so you can easily see how e
 
     <tr>
       <td>
-        Organize everything you monitor and provide context for your entities.
+        Visualize the aggregated health and activity data from all you monitor.  
       </td>
 
       <td>
-        Add [tags](/docs/tagging-use-tags-organize-group-what-you-monitor) to all your entities.
+        Group and monitor any entities together into functional team-focused, project-focused groupings, or any other attribute, with [workloads](/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-workloads-create-groupings-entities). 
       </td>
     </tr>
 
     <tr>
       <td>
-        Understand how everything is connected via API.
+        Provide context for your entities.
       </td>
 
       <td>
-        The [NerdGraph GraphiQL explorer](/docs/apis/graphql-api/getting-started/introduction-new-relic-graphql-api) manages all your entities, tags, and relationships.
+        Add [tags](/docs/tagging-use-tags-organize-group-what-you-monitor) to all your entities. Or [create tags](/docs/apis/graphql-api/tutorials/graphql-tagging-api-tutorial) for teams and all the services they monitor. Use tags to illustrate relationships and contextual information for what you monitor. By thoughtfully tagging your entities, you can connect all the data your teams need to understand their increasingly complex and interdependent systems.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Create visuals that showcase your business needs at a glance. 
+      </td>
+
+      <td>
+        Tailor [custom dashboards](#dashboards) for your unique needs. 
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Fetch and analyze specific data.
```

Comment:
I'd put the query builder above dashboards, because you need to build queries to get those dashboards working, right?

Reply:
Yep!